packages/ui/src/AttendanceSheet/utils.ts

```diff
@@ -8,24 +8,19 @@ enum BookingDuration {
   "2+h" = "2H+",
 }
 
-export const calculateIntervalDuration = (
-  startTime: string,
-  endTime: string
-): BookingDuration => {
-  const luxonStart = DateTime.fromISO(startTime);
-  const luxonEnd = DateTime.fromISO(endTime);
-  const { minutes } = luxonEnd.diff(luxonStart, ["minutes"]);
+export const getIntervalString = (interval: string): BookingDuration => {
+  const durationInMinutes = calculateIntervalDurationInMinutes(interval);
```

Comment:
Let's not use `getIntervalString` here. That would imply you'd get something like: `"18:00-19:00"` string.
Maybe call this `getDurationLabel` or `getIntervalLabel` if you want to change it from `calculateIntervalDuration` (and I see as point in doing so)